packages/framework/src/DataFixtures/Demo/OrderDataFixture.php

```diff
@@ -112,6 +112,7 @@ public function load(ObjectManager $manager)
         $orderData->domainId = Domain::FIRST_DOMAIN_ID;
         $orderData->currency = $this->getReference(CurrencyDataFixture::CURRENCY_CZK);
         $orderData->createdAt = $this->faker->dateTimeBetween('-1 week', 'now');
+        $orderData->createdAsAdministrator = $this->getReference(AdministratorDataFixture::ADMINISTRATOR);
```

Comment:
Some could be created as superadmin instead, to add more special cases.

Also, the reason behind this change should be described in the commit msg and the PR description.

Reply:
I will add info to cm that it was never used

Reply:
done